web/impact/impact/tests/test_application_detail_view.py

```diff
@@ -0,0 +1,61 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import json
+from jsonschema import Draft4Validator
+
+from django.urls import reverse
+from impact.tests.factories import ApplicationFactory
+from impact.tests.api_test_case import APITestCase
+from impact.tests.utils import assert_fields
+from impact.v1.views import ApplicationDetailView
+
+APPLICATION_GET_FIELDS = [
+    "id",
+    "created_at",
+    "updated_at",
+    "application_status",
+    "cycle",
+    "startup",
+    "application_type",
+]
+
+
+class TestApplicationDetailView(APITestCase):
+    def test_get(self):
+        application = ApplicationFactory()
+        with self.login(email=self.basic_user().email):
+            url = reverse(ApplicationDetailView.view_name,
+                          args=[application.id])
+            response = self.client.get(url)
+            assert response.data["created_at"] == application.created_at
```

Comment:
I know there are many other tests which use raw `assert a == b`, but I prefer that we  use `assertEqual` wherever possible. One benefit is that, when it fails, it tells you what the values are. Plain `assert` does not do that, producing more work. It's acceptable for "this will never fail" preconditions not directly tied to the object of the test, but that's about it.